app/views/dashboard/_discovery_widget.html.erb

```diff
@@ -4,7 +4,7 @@
 <% if hosts.empty? %>
    <p class="ca"><%= _("No discovered hosts available") %></p>
 <% else %>
-    <%= render :partial => 'discoverey_widget_host_list', :locals => { :hosts => hosts } %>
+    <%= render :partial => 'discovery_widget_host_list', :locals => { :hosts => hosts } %>
```

Comment:
please double check the widget size, in my local testing it was bigger then its allowed size.